Program.cs

```diff
@@ -1,2 +1,189 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using CalculatorLibrary;
+using System.Text.RegularExpressions;
+
+namespace CalculatorProgram
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool endApp = false;
+
+            DisplayTitle();
+
+            Calculator calculator = new Calculator();
```

Comment:
You can write less code here: var calculator = new Calculator(); or Calculator calculator = new()